app/models/supported_ssg.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+# rubocop:disable Metrics/BlockLength
+SupportedSsg = Struct.new(:id, :package, :version, :profiles,
+                          :os_major_version, :os_minor_version,
+                          keyword_init: true) do
+  self::SUPPORTED_FILE = Rails.root.join('config/supported_ssg.yaml')
+
+  class << self
```

Comment:
For the next part of this work, we will need to index into this collection by RHEL minor version and get the supported SSG version. We will need to implement `[]` or `find`. We can add it now or later, up to you.

Reply:
Yes, I have that in mind.